heartbeat/sender/peerAuthenticationSender_test.go

```diff
@@ -152,15 +157,14 @@ func TestNewPeerAuthenticationSender(t *testing.T) {
 		args := createMockPeerAuthenticationSenderArgs()
 		sender, err := NewPeerAuthenticationSender(args)
 
-		assert.NotNil(t, sender)
+		assert.False(t, check.IfNil(sender))
```

Comment:
can we change all the `assert.Nil(t, sender)` to `assert.True(t, check.IfNil(sender))` from the above tests?